evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -120,6 +124,24 @@ fn sign(private_key: U256, digest: H256, chain_id: U256) -> Result {
     Ok((sig.v, r_bytes, s_bytes).encode().into())
 }
 
+fn create_wallet(private_key: U256, label: Option<String>, state: &mut Cheatcodes) -> Result {
+    let key = parse_private_key(private_key)?;
+    let addr = utils::secret_key_to_address(&key);
+
+    let pub_key = key.verifying_key().as_affine().to_encoded_point(false);
+    let pub_key_x = pub_key.x().ok_or("No x coordinate was found")?;
+    let pub_key_y = pub_key.y().ok_or("No y coordinate was found")?;
+
+    let pub_key_x = U256::from(pub_key_x.as_slice());
+    let pub_key_y = U256::from(pub_key_y.as_slice());
```

Comment:
This is the best way I could find to extract the x and y coordinates. I'm not extremely familiar with the details of ECDSA algorithms, so I'm not 100% sure if this is accurate either.